### ISSUE TITLE
Figure 1.2 - minor typos

### ISSUE THREAD

**Issue by sabuncumurat** [PacktPublishing/Infrastructure-as-Code-for-Beginners-#1]
Deployment 2 column: The Network Interface box needs to be green.
Deployment 3 column: The IP address and VM boxes need to be labeled as Success.

![2024-04-28 131032 - FoxitPDFReader](https://github.com/PacktPublishing/Infrastructure-as-Code-for-Beginners/assets/91539510/6c2eac92-1841-432c-9244-de0ac38faf17)

**Comment by russmckendrick** [comment]
Good catch, @sabuncumurat. Thank you for reporting - I have attached the updated version and will submit it to the publishers to update.

![B19537_01-B19537_01_02](https://github.com/PacktPublishing/Infrastructure-as-Code-for-Beginners/assets/121128/4f2ece6d-e7b3-4e06-a6a7-24004891d913)
